tools/pbxproj_generators/lib/PBXProj/src/Identifiers.swift

```diff
@@ -0,0 +1,7 @@
+public enum Identifiers {
```

Comment:
I need to add a doc comment to this, because it's not clear why this is needed at first glance.